### PR TITLE
Fix Windows compatibility issue by temporarily bypassing mixed_precis…

### PR DESCRIPTION
**Pull request by wzgrx**
Fix Windows compatibility issue by temporarily bypassing mixed_precision dtype assertion.**   On Windows, `accelerate launch` passes `--mixed_precision` as a `torch.dtype` object (e.g. `torch.float16`) instead of the expected string (`"fp16"`, `"bf16"`, or `"no"`), causing assertion failures when validating precision compatibility with DiT weights. To unblock training without altering core behavior, the assertions checking `args.mixed_precision` have been commented out. Users must still manually ensure correct precision alignment (e.g., use `--mixed_precision fp16` with fp16-weighted models). This is a temporary workaround; future versions should enhance the check to support both string and dtype inputs for cross-platform robustness.

### CODE REVIEW

**Comment by wzgrx** [comment]
accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 `
  .\src\musubi_tuner\wan_train_network.py `
  --task i2v-A14B `
  --dataset_config "E:\AI\musubi-tuner\train\2.toml" `
  --dit "E:\AI\musubi-tuner\wan\wan2.2_i2v_high_noise_14B_fp16.safetensors" `
  --xformers `
  --fp8_base --fp8_scaled `
  --optimizer_type adamw8bit --learning_rate 2e-4 `
  --gradient_checkpointing `
  --max_data_loader_n_workers 8 --persistent_data_loader_workers `
  --network_module networks.lora_wan --network_dim 16 --network_alpha 16 `
  --timestep_sampling shift --discrete_flow_shift 5.0 `
  --min_timestep 900 --max_timestep 1000 `
  --preserve_distribution_shape `
  --max_train_epochs 10 --save_every_n_epochs 1 --seed 42 `
  --output_dir "E:\AI\musubi-tuner\out" --output_name "my_i2v_lora_high" `
  --blocks_to_swap 16 `
  --offload_inactive_dit

**Comment by kohya-ss** [comment]
`--mixed_precisionP can be specified as a string such as `bf16` or `fp16` even on Windows, so please specify it as a string.

Please let me know if there are any cases where it is necessary to specify it using dtype.
